inspirehep/dojson/hep/fields/bd01x09x.py

```diff
@@ -100,7 +100,7 @@ def isbns(self, key, value):
     elif 'ebook' == b_in_marc:
         medium = 'online'
     elif 'paperback' in b_in_marc:
-        medium = 'online'
+        medium = 'hardcover'
```

Comment:
Of course, I typed the wrong thing there https://github.com/inspirehep/inspire-next/pull/1910#discussion_r100781702